packages/bezier-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -25,15 +35,6 @@
     }
   }
 
-  display: flex;
-  align-items: center;
-  justify-content: center;
-
-  margin: 1px;
-
-  background-color: var(--bg-white-normal);
-  border-radius: 7px;
-  box-shadow: inset 0 0 0 2px var(--bdr-black-dark);
 
```

Comment:
```suggestion
```

Do you have prettier on extension? It should be auto formatted on save. 

There is blank space on L14 as well. 

<img width="153" alt="image" src="https://github.com/user-attachments/assets/e8cce147-75c9-4f2b-aef5-b8214fb247dc">

Reply:
I was pretty sure prettier ran. I could've missed it. 
Fixed it.